doppio/commands/utils.py

```
@@ -39,22 +47,27 @@ def add_commands_to_root_package_json(app, spa_name):
 	with package_json_path.open("w") as f:
 		json.dump(data, f, indent=2)
 
-	# Update app's package.json
 	app_package_json_path: Path = app_path / "package.json"
 
 	if not app_package_json_path.exists():
 		subprocess.run(["npm", "init", "--yes"], cwd=app_path)
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)